root/tree/readcin/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-ROOTTEST_GENERATE_EXECUTABLE(readFromCin readFromCin.cxx LIBRARIES Core Hist RIO Net Graf Graf3d Gpad Tree Rint Postscript Matrix Physics MathCore Thread MultiProc)
+if(MSVC)
+    ROOTTEST_GENERATE_EXECUTABLE(readFromCin readFromCin.cxx LIBRARIES Core Hist RIO Net Graf Graf3d Gpad Tree Rint Postscript Matrix Physics MathCore Thread)
+else()
+    ROOTTEST_GENERATE_EXECUTABLE(readFromCin readFromCin.cxx LIBRARIES Core Hist RIO Net Graf Graf3d Gpad Tree Rint Postscript Matrix Physics MathCore Thread MultiProc)
```

Comment:
Would it be worth using a ```${OptionalMultiProc}``` variable in multiple place rather than ```if (MSCV)``` in those same places?

Reply:
I'll check, but `MultiProc` is definitively not supported on Windows (and I'm not going to do anything about it)

Reply:
Humm what I am proposing:
```
ROOTTEST_GENERATE_EXECUTABLE(readFromCin readFromCin.cxx LIBRARIES Core Hist RIO Net Graf Graf3d Gpad Tree Rint Postscript Matrix Physics MathCore Thread ${OptionalMultiProc} )
```
with ${OptionalMultiProc} being a ```empty``` on windows and ```MultiProc``` everywhere else.

Reply:
OK, will do that, thanks for the suggestion

Reply:
Done